src/explore-education-statistics-frontend/src/modules/find-statistics/components/MobileFilters.tsx

```diff
@@ -0,0 +1,51 @@
+import Button from '@common/components/Button';
+import ButtonText from '@common/components/ButtonText';
+import Modal from '@common/components/Modal';
+import { ThemeSummary } from '@common/services/themeService';
+import { ReleaseType } from '@common/services/types/releaseType';
+import AdvancedFilters from '@frontend/modules/find-statistics/components/AdvancedFilters';
+import { FilterChangeHandler } from '@frontend/modules/find-statistics/components/Filters';
+import styles from '@frontend/modules/find-statistics/components/MobileFilters.module.scss';
+import ThemeFilters from '@frontend/modules/find-statistics/components/ThemeFilters';
+import React from 'react';
+
+interface Props {
+  releaseType?: ReleaseType;
+  themeId?: string;
+  themes: ThemeSummary[];
+  totalResults: number;
+  onChange: FilterChangeHandler;
+  onClose: () => void;
+}
+
+const MobileFilters = ({
+  releaseType,
+  themeId,
+  themes,
+  totalResults,
+  onChange,
+  onClose,
+}: Props) => {
+  const totalResultsString = `${totalResults} ${
+    totalResults !== 1 ? 'results' : 'result'
+  }`;
+
+  return (
+    <Modal fullScreen hideTitle title="Filter publications" onExit={onClose}>
+      {/* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */}
+      <div className={styles.container} tabIndex={0}>
```

Comment:
Does this need to be tabbable? It's not great in terms of the keyboard experience as the entire modal shows a focus on rendering. Upon tabbing again, we get a really faint second outline:

![image](https://user-images.githubusercontent.com/9917868/205954023-6d3a9ecc-8e81-4eab-bb36-ed57f7454b77.png)

I actually didn't notice this outline initially so it was like the Tab press had been swallowed by some invisible element.

Would suggest we remove this so there's only a single focus when the modal renders. Otherwise, it's a little annoying to have to Tab twice to reach the actual content.

If this is related to the scroll container, would suggest that we also make the modal itself scrollable when it's in full screen mode.

Reply:
yeah you're right, I've taken that our and made the fullscreen modal scrollable.